generatederrors.go

```diff
@@ -22,90 +22,90 @@ package edgedb
 import "fmt"
 
 const (
+	ShouldRetry     ErrorTag = "SHOULD_RETRY"
```

Comment:
Are these changes related to the PR?

Reply:
Yes. The README says you can format the code using `make format` but that file was never formatted. I figured it would be strange for someone to run `make format` in a fresh checkout and get a huge diff. I also added `make format` to the error generation so that happens automatically in the future.